salsah/src/public/vendor/ckeditor-4.7.2-Editor/_config.js

```diff
@@ -0,0 +1,46 @@
+/**
```

Comment:
Did you rename this file?

I get this error: `"NetworkError: 404 Not Found - http://localhost:3335/vendor/ckeditor-4.7.2-Editor/config.js?t=H7HD"`

You could simply overwrite the function `CKEDITOR.editorConfig` in `jquery.htmleditor,js`, no? Of course we would have to make sure that all ckeditor files are loaded before our JS is executed.

Otherwise we will have to change this on every version update.

Reply:
will do (I renamed the file to make sure it is not loaded)
- [ ] get rid of 404 for file `config.js`

Reply:
done in b8aece417c4bf55ab807df34a071dae3bf9b5369

ckeditor still looks for the file `config.js` when defining a `CKEDITOR.editorConfig = function( config ) {` in `jquery.htmleditor.js` (it is executed after `CKEDITOR.replace` and before the `CKEDITOR.editorConfig` we define in `jquery.htmleditor.js`.

So I load a custom empty config that will not disturb our config and won't be overwritten at the next ckeditor install.